test/support/integration/teams_tests.ex

```diff
@@ -0,0 +1,177 @@
+defmodule Livebook.TeamsIntegrationHelper do
+  alias Livebook.{Factory, Hubs, Teams, TeamsRPC}
+
+  import ExUnit.Assertions
+  import Phoenix.ConnTest
+
+  @endpoint LivebookWeb.Endpoint
+
+  def workspace(context) do
+    case {context[:workspace_for], context[:workspace_persisted]} do
```

Comment:
"Workspace" is a user-facing naming, but I think everywhere in the code we still say "hub", so I would keep that naming consistent. What do you think?

In this case it seems you only need to rename `:workspace`, `workspace_for` and `workspace_persisted`, so it should be a simple replace.

Reply:
Sounds good :)